src/popup/router/components/RecentTransactions.vue

```diff
@@ -147,7 +147,7 @@ export default {
   .message,
   .spinner {
     min-height: 174px;
-    padding-bottom: 8px;
+    padding-bottom: 48px;
     flex-grow: 1;
     display: flex;
     align-items: center;
```

Comment:
I still have misaligned `spinner` and `message` in desktop version
![image](https://user-images.githubusercontent.com/7098449/118755217-d3f68900-b8ab-11eb-9775-03827de5754d.png)
After removing this styles
```suggestion
```
`message` and `spinner` are aligned on the top which is looks nice everywhere for me:
![image](https://user-images.githubusercontent.com/7098449/118755391-1cae4200-b8ac-11eb-9166-0a7564c9e8b7.png)
Maybe let's fix this way?

Reply:
I've re-implemented it as per figma designs where elements are centered. It works on all platforms now.

Reply:
![Screenshot 2021-05-20 at 14 30 00](https://user-images.githubusercontent.com/482351/118971446-1daca580-b978-11eb-9896-fa35ddefe6eb.png)
<img src="https://user-images.githubusercontent.com/482351/118971451-1f766900-b978-11eb-93be-7fc041364e19.png" width="360" />